### DISCUSSION TITLE
Remove `<lastmod>`

### DISCUSSION THREAD

**Discussion by BenSotty** [spatie/laravel-sitemap#509] [Q&A]
Hi there 👋,

For some pages I do not know when they have been modified.
So, for those pages I don't want the `<lastmod>` to appear in the sitemap.xml.
But for  a `Spatie\Sitemap\Tags\Url`, `setLastModificationDate` expects a `DateTimeInterface` as argument.
Thus: `$url->setLastModificationDate(null);` breaks.
Is there a way to remove `<lastmod>` from the sitemap ?

Thanks a lot for your help.
Thanks a lot for the tool :)

**Comment by ideea** [comment]
any update on this one? :)
